packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { IButtonStyles } from '../Button.Props';
+import {
+  ITheme,
+  getTheme,
+  mergeStyles,
+  mergeStyleSets
+} from '../../../Styling';
+import {
+  getStyles as getDefaultButtonStyles
+} from '../DefaultButton/DefaultButton.styles';
+
+export function getStyles(
```

Comment:
I'm not really sure. Styles are a function of the theme and of customStyles.

It is possible that we can pull the "global" theme, but it removes the ability to override the theme in a context.

It is possible to not take in customStyles, and to instead mix them in from within the component itself.